R/database.R

```diff
@@ -227,6 +227,20 @@ get_ssm_by_samples = function(these_sample_ids,
           verbose = FALSE,
           ssh_session = ssh_session
         )
+
```

Comment:
I think this method would throw an error if the user opted for `basic_columns = FALSE` since this doesn't specify `col_types` for columns beyond the first ~45. 

I think the place to set this is in the `fread_maf` function. It already has a few `col_types` specified explicitly, but it would need to be extended to cover all columns in case of an empty or near-empty file. `fread_maf` uses `fread`, @rdmorin should it be updated to use `read_tsv`? One of the handy things about `read_tsv` is that you can load the column specifications for a good maf file, save that as an R object, then use that to specify the column types in a `read_tsv` function. There's a useful example [here](https://readr.tidyverse.org/reference/spec.html#ref-examples).

Reply:
Thanks for your comment. fread_maf has now been updated (and get_ssm_by_samples reverted) to hold column types for all 116 maf-columns.